ember-app/app/services/filter_groups/milestone.js

```diff
@@ -1,29 +1,50 @@
 import Ember from 'ember';
 
+function attr(modelProp, map) {
+  return Ember.computed("model." + modelProp, {
+    get: function(key){
+      var filters = this.get("model." + modelProp).map(map);
+      filters.insertAt(0, Ember.Object.create({
```

Comment:
Yep, it's inserted every time the filters are recomputed, but it makes sense because the whole array is being mapped every time the model changes